### PR TITLE
feat: UILD-409: Write installation documentation for ui-linked-data module

### PR DESCRIPTION
**Pull request by s3fs**
https://folio-org.atlassian.net/browse/UILD-409

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,51 +6,35 @@ This software is distributed under the terms of the Apache License, Version 2.0.
...
 
-## Dependencies
-This module requires the following components to function:
+## Development
 
-1. [`@folio-org/linked-data`](https://github.com/folio-org/ui-linked-data) (also known as `ui-linked-data`, `linked-data`). This module needs to be built as a library in order for `ui-bf-folio-wrapper` to work. Refer to point 1 from its [Usage: As an embedded application](https://github.com/folio-org/ui-linked-data?tab=readme-ov-file#as-an-embedded-application) section on how to do so.
+If you would like to run a development build of this module while making changes to the [ui-linked-data](https://github.com/folio-org/ui-linked-data) within it, there are a few steps to take:
 
-## Prerequisites
+0.
```
Comment (SKarolFolio) [inline README.md:17]: Do we need this empty step?

**Review by SKarolFolio — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -6,51 +6,34 @@ This software is distributed under the terms of the Apache License, Version 2.0.
```
Comment (pkjacob) [inline README.md:6]: Line 1 contains the text "ui-linked-data". It has to be changed to "ui-ld-folio-wrapper"

File: README.md [hunk 3]
```diff
@@ -6,51 +6,34 @@ This software is distributed under the terms of the Apache License, Version 2.0.
 
 ## Introduction
 
-Congratulations on creating a new Stripes UI app module!  Follow the instructions below to run ui-linked-data and start your development.
+This is a [Stripes](https://github.com/folio-org/stripes-core/) UI module for handling operations with Linked data resources.
```
Comment (pkjacob) [inline README.md:9]:
```suggestion
This is a [Stripes](https://github.com/folio-org/stripes-core/) UI module for performing operations on Linked data resources.
```

Comment (pkjacob) [inline README.md:9]:
```suggestion
This is a [Stripes](https://github.com/folio-org/stripes-core/) UI module designed for performing operations on a library's Linked Data resources.
```

File: README.md [hunk 4]
```diff
@@ -6,51 +6,34 @@ This software is distributed under the terms of the Apache License, Version 2.0.
...
 
-TODO: Modify this README to replace these sections about getting started.
+This module acts as an adapter for [ui-linked-data](https://github.com/folio-org/ui-linked-data) module. Essentially, most of the functionality of this module is defined there.
```
Comment (pkjacob) [inline README.md:11]: This module serves as an adapter for the [ui-linked-data](https://github.com/folio-org/ui-linked-data) application. Essentially, most of the functionality of this module is defined in [ui-linked-data](https://github.com/folio-org/ui-linked-data). This module enables the [ui-linked-data](https://github.com/folio-org/ui-linked-data) application to run seamlessly within the FOLIO platform.

File: README.md [hunk 5]
```diff
@@ -6,51 +6,34 @@ This software is distributed under the terms of the Apache License, Version 2.0.
...
+## Additional information
 
-## Run your new app
+See the related [ui-linked-data](https://github.com/folio-org/ui-linked-data) module.
 
-Run the following from the ui-linked-data directory to serve your new app using a development server:
-```
-stripes serve
-```
+Other [modules](https://dev.folio.org/source-code/#client-side).
```
Comment (pkjacob) [inline README.md:34]: Eventhough the hyperlink says "modules", the link is not pointing to any modules.

File: README.md [hunk 6]
```diff
@@ -6,51 +6,34 @@ This software is distributed under the terms of the Apache License, Version 2.0.
...
+    ```
+4. Perform the code changes you need.
+5. Build the [ui-linked-data](https://github.com/folio-org/ui-linked-data) module as a library. The steps are described [here](https://github.com/folio-org/ui-linked-data?tab=readme-ov-file#as-an-embedded-application).
+6. After the [ui-linked-data](https://github.com/folio-org/ui-linked-data) module finishes its build, the Stripes build should pick up the changes and reload itself.
 
-In order to view and log into the platform being served up, a suitable Okapi backend will need to be running. The [testing-backend](https://app.vagrantup.com/folio/boxes/testing-backend) Vagrant box should work if your app does not yet have its own backend module.
+## Additional information
 
-## Run your new app
+See the related [ui-linked-data](https://github.com/folio-org/ui-linked-data) module.
```
Comment (pkjacob) [inline README.md:32]:
```suggestion
## Additional information
See the related [ui-linked-data](https://github.com/folio-org/ui-linked-data) module.
Backend module corresponding to this UI module: [mod-linked-data](https://github.com/folio-org/mod-linked-data)
```

File: README.md (hunk 5 above)
Comment (s3fs) [reply README.md:34]: This is a common section for UI modules. Check here: https://github.com/folio-org/ui-users/?tab=readme-ov-file#additional-information or here: https://github.com/folio-org/ui-users/?tab=readme-ov-file#additional-information

**Review by pkjacob — APPROVED** [review]

**Pull request merged**